p4rt_app/sonic/packetio_impl_test.cc

```diff
@@ -147,27 +149,9 @@ TEST(PacketIoImplTest, AddingDuplicatePacketIoPortsIsANoop) {
   EXPECT_TRUE(packetio_impl.IsValidPortForReceive("Ethernet1/1"));
 }
 
-TEST(PacketIoImplTest, NoActionOnAddingNonSdnPacketIoPorts) {
-  auto mock_call_adapter = absl::make_unique<sonic::MockSystemCallAdapter>();
-
-  // We should not create any sockets since the port is ignored.
-  EXPECT_CALL(*mock_call_adapter, socket).Times(0);
-  EXPECT_CALL(*mock_call_adapter, setsockopt).Times(0);
-  EXPECT_CALL(*mock_call_adapter, if_nametoindex).Times(0);
-  EXPECT_CALL(*mock_call_adapter, bind).Times(0);
-
-  PacketIoImpl packetio_impl(std::move(mock_call_adapter),
-                             PacketIoOptions{
-                                 .callback_function = EmptyPacketInCallback,
-                             });
-  ASSERT_OK(packetio_impl.AddPacketIoPort("loopback0"));
-
-  // Checks that ports are not valid for transmit and receive.
-  EXPECT_FALSE(packetio_impl.IsValidPortForTransmit("loopback0"));
-  EXPECT_FALSE(packetio_impl.IsValidPortForReceive("loopback0"));
-}
-
 TEST(PacketIoImplTest, AddAndRemoveMultiplePacketIoPorts) {
+  constexpr absl::string_view kSubmitToIngress = "send_to_ingress";
```

Comment:
How did this test pass with a different name ?

Reply:
@kishanps, In PacketIoImplTest.AddAndRemoveMultiplePacketIoPorts(), it calls packetio_impl.IsValidPortForTransmit() which validates the port name using port_to_socket_ which is a flat_hash_map. I dumped the contents of hash_map which had the following entries.

**LOGS:**
I0514 15:23:09.659754    12 packetio_impl.cc:136] divya::port_to_socket_.first Ethernet1/1
I0514 15:23:09.659758    12 packetio_impl.cc:136] divya::port_to_socket_.first send_to_ingress.

Since we have send_to_ingress, test case is passing.